.buildbot.sh

```diff
@@ -29,10 +31,10 @@ cargo fmt --all -- --check
 cd ykllvm
 ykllvm_hash=$(git rev-parse HEAD)
 cached_ykllvm=0
-if [ -f /opt/ykllvm_cache/ykllvm-release-with-assertions-${ykllvm_hash}.tgz ]; then
+if [ -f /opt/ykllvm_cache/ykllvm-release-with-assertions-"${ykllvm_hash}".tgz ]; then
```

Comment:
Go on then. Why is this quoting better?

Reply:
It stops word splitting. Which I didn't even realise!